lib/components/form/Details.tsx

```diff
@@ -146,23 +145,40 @@ const Details: FC<Props> = ({ lastPageCallback, nextPageCallback, site }) => {
           />
 
           <Input
-            label={
-              <>
-                Solar panel capacity
-                <span className="text-xs font-normal"> (optional)</span>
-              </>
+            id="solar-panel-tilt"
+            label="Inverter capacity"
+            value={String(panelDetails.inverterCapacityKw)}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              setFormData({
+                ...panelDetails,
+                inverterCapacityKw: e.currentTarget.value,
+              })
             }
+            inputProps={{
+              type: 'number',
+              placeholder: '3000 kW',
+              min: '0',
+              step: 'any',
+              required: true,
+              onKeyDown: preventMinus,
+              pattern: '[0-9]*',
+              inputMode: 'numeric',
+            }}
+          />
+
+          <Input
+            label="Solar panel nameplate capacity"
             id="solar-panel-capacity"
-            value={String(panelDetails.capacity)}
+            value={String(panelDetails.moduleCapacityKw)}
             onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
               setFormData({
                 ...panelDetails,
-                capacity: e.currentTarget.value,
+                moduleCapacityKw: e.currentTarget.value,
               })
             }
             inputProps={{
               type: 'number',
-              placeholder: '2800 kW',
+              placeholder: '3000 kW',
```

Comment:
lol we should probably lower this placeholder—5 kW seems much more reasonable for the scale of home arrays